vpr/src/draw/ui_setup.cpp

```diff
@@ -174,7 +174,21 @@ void hide_crit_path_button(ezgl::application* app) {
     hide_widget("CritPathLabel", app);
     hide_widget("ToggleCritPath", app);
 }
-
+/*
+ * @brief hides or displays critical path routing / routing delay
```

Comment:
Add at end: "UI elements,  Use to ensure we don't show inactive buttons etc. when routing data doesn't exist."

Reply:
Can just put this in the .h file.